### ISSUE TITLE
Having trouble running the AIAA-OHIF interagtion with clara train v4.1. Is it possible?

### ISSUE THREAD

**Issue by jbobeda** [NVIDIA/clara-train-examples#16]
Hi!

I'm following the [tutorial in clara-train-examples](https://github.com/NVIDIA/clara-train-examples/blob/master/PyTorch/NoteBooks/AIAA/AIAAwOHIF.ipynb) and I managed to get it to work with Ohif and clara train v4.0.
My objective now is to make it work with a custom Tensorflow algorithm, that should be possible as per [this forum topic](https://forums.developer.nvidia.com/t/clara-train-4-0-aiaa-tensorflow-models-still-supported/195152). Following the Clara Train SDK documentation I should approach this from the v4.1 model, as from this version on the custom models must follow the [MONAI Label App template](https://docs.monai.io/projects/label/en/latest/modules.html).
My problem arises when using this version of the SDK as backend for the OHIF-AIAA integration in [clara-train-examples](https://github.com/NVIDIA/clara-train-examples/tree/master/PyTorch/NoteBooks/AIAA/OHIF-Orthanc), even with the models already available in the NGC registry. I load the model clara_pt_spleen_ct_segmentation, I click the "Run Segmentation" button and in OHIF's frontend I get the following events:

  - Please wait while creating new AIAA Session! -> AIAA Data preparation complete -> Failed to Create AIAA Session... Reason: undefined

I've tried to debug the issue using Chrome's Developer tools and found out that OHIF receives a **"422 Unprocessable Entity"** when sending the following call to the SDK: **curl -X 'PUT' 'http://MY_SERVER:5000/session/?expiry=0&save_as=.nii'**. I've checked if the same call is also sent when using the v4.0 SDK and it seems to be exactly the same. In my opinion the OHIF-AIAA integration is not able to process back the response from the API, that changes between versions. Here two responses I get doing by myself calls to that endpoint in the SDK:

  - v4.0: {"session_id": "dacc097c-e266-11ec-a4aa-0242ac160003"}
  - v4.1: {"session_id":"5558a0de-e266-11ec-93c5-0242ac150003","session_info":{"name":"5558a0de-e266-11ec-93c5-0242ac150003","path":"/root/.cache/monailabel/sessions/5558a0de-e266-11ec-93c5-0242ac150003","image":"/root/.cache/monailabel/sessions/5558a0de-e266-11ec-93c5-0242ac150003/my_dicom.dcm","meta":{},"create_ts":1654168968,"last_access_ts":1654168968,"expiry":3600}

Could you please confirm if the problem comes indeed from a version mismatch between the OHIF-AIAA extension and Clara Train SDK v4.1?
If so, do you have any plans to update the extension so it works with the newest SDK versions?

Thanks for your time,

Javier
